pickem/pickem_homepage/views.py

```
@@ -40,6 +71,121 @@ def is_commissioner(user):
+            usernames[key] = "Unknown User"
+        else:
+            usernames[key] = f"User {key}"
+
+        account = social_accounts.get(int(key)) if key.isdigit() else None
+        if account:
+            avatars[key] = account.get_avatar_url()
+        elif user:
+            email_hash = hashlib.md5((user.email or '').lower().encode('utf-8')).hexdigest()
+            avatars[key] = f"https://www.gravatar.com/avatar/{email_hash}?d=identicon&s=64"
+        else:
+            avatars[key] = f"https://www.gravatar.com/avatar/?d=identicon&s=64"
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)